Dockerfile.charterafrica

```diff
@@ -59,6 +59,7 @@ ARG NEXT_TELEMETRY_DISABLED=1 \
     PAYLOAD_PUBLIC_APP_URL=http://localhost:3000 \
     PAYLOAD_SECRET_KEY
 
+RUN cd apps/charterafrica && npx payload migrate && cd ../../
```

Comment:
🤔 ... Should running migration be part of docker build?

Reply:
@kilemensi should this be independent of the build? e.g in the CI pipeline?. But I think they should be run before next-build.

Reply:
Running migrations should not be part of the build process at all @koechkevin. Imagine you're building your image and migrations run but the next step after migration fails. You now can't deploy the app but at the same time, the schema has already been updated so the current running up is likely to fail.

... running migrations should be part of deployment strategy i.e. you're 100% sure you have deployable image.